src/acceleration/BaseQNAcceleration.cpp

```diff
@@ -115,12 +115,11 @@ void BaseQNAcceleration::initialize(
     }
   }
 
+  _matrixCols.clear();
```

Comment:
If I move these calls in some reset function, then the initialization is lacking the context of which vector is actually empty or not.
While nice in theory, this will lead to a lot of jumping back and forth, especially as the QN base has so many members.

Reply:
We concluded to clean this up when we do the next refactoring pass of the Acceleration hierarchy.